use std::io::Error;

// pub fn new(args: Vec<String>) -> Result<(), Error> {
//     if args.len() <
// }
